test/katex-spec.js

```diff
@@ -1251,6 +1251,11 @@ describe("A begin/end parser", function() {
     it("should allow \\cr as a line terminator", function() {
         expect("\\begin{matrix}a&b\\cr c&d\\end{matrix}").toParse();
     });
+
+    it("should eat a final newline", function() {
+        const m3 = getParsed("\\begin{matrix}a&b\\\\ c&d \\end{matrix}")[0];
```

Comment:
Should the TeX code be:
```
\\begin{matrix}a&b\\\\ c&d \\\\ \\end{matrix}
```
to include the extra newline that's supposed to get eaten?

Reply:
Right you are. I'll make the change and resubmit.